src/web/jdv/client/build.sh

```diff
@@ -8,10 +8,6 @@ pushd "$DIR"
 output_path='dist'
 [[ ! -z "$1" ]] && output_path="$1"
 
-pushd ../../
```

Comment:
Could this cause an error for someone only running/launching the JDV?

Reply:
No, this was a duplicate installation of the dependencies.

The top level run.sh in the web/jdv folder calls the same install_dependencies.sh that web/runs.sh calls.

The build doesn't need to install the messages twice because the python deb package is setup to create a virtual python environment.